homeassistant/components/mobile_app/webhook.py

```diff
@@ -0,0 +1,162 @@
+"""Webhook handlers for mobile_app."""
+from functools import partial
+import logging
+from typing import Dict
+
+from aiohttp.web import HTTPBadRequest, json_response, Response, Request
+import voluptuous as vol
+
+from homeassistant.components.device_tracker import (DOMAIN as DT_DOMAIN,
+                                                     SERVICE_SEE as DT_SEE)
+from homeassistant.components.webhook import async_register as webhook_register
+
+from homeassistant.const import (ATTR_DOMAIN, ATTR_SERVICE, ATTR_SERVICE_DATA,
+                                 CONF_WEBHOOK_ID, HTTP_BAD_REQUEST)
+from homeassistant.core import EventOrigin
+from homeassistant.exceptions import (HomeAssistantError, ServiceNotFound,
+                                      TemplateError)
+from homeassistant.helpers import template
+from homeassistant.helpers.discovery import load_platform
+from homeassistant.helpers.storage import Store
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import (ATTR_APP_COMPONENT, DATA_DELETED_IDS,
+                    ATTR_DEVICE_NAME, ATTR_EVENT_DATA, ATTR_EVENT_TYPE,
+                    DATA_REGISTRATIONS, ATTR_TEMPLATE, ATTR_TEMPLATE_VARIABLES,
+                    ATTR_WEBHOOK_DATA, ATTR_WEBHOOK_ENCRYPTED,
+                    ATTR_WEBHOOK_ENCRYPTED_DATA, ATTR_WEBHOOK_TYPE,
+                    CONF_SECRET, DOMAIN, WEBHOOK_PAYLOAD_SCHEMA,
+                    WEBHOOK_SCHEMAS, WEBHOOK_TYPE_CALL_SERVICE,
+                    WEBHOOK_TYPE_FIRE_EVENT, WEBHOOK_TYPE_RENDER_TEMPLATE,
+                    WEBHOOK_TYPE_UPDATE_LOCATION,
+                    WEBHOOK_TYPE_UPDATE_REGISTRATION)
+
+from .helpers import (_decrypt_payload, empty_okay_response,
+                      registration_context, safe_registration, savable_state)
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def register_deleted_webhooks(hass: HomeAssistantType, store: Store):
```

Comment:
You're still passing around store but it's also stored in `hass.data`. No biggy, no need to change